packages/six.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Six < Package
+  description 'Six is a Python 2 and 3 compatibility library.'
+  homepage 'https://github.com/benjaminp/six'
+  version '1.11.0'
+  source_url 'https://github.com/benjaminp/six/archive/1.11.0.tar.gz'
+  source_sha256 '927dc6fcfccd4e32e1ce161a20bf8cda39d8c9d5f7a845774486907178f69bd4'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'python3'
+  depends_on 'python27'
+
+  def self.build
+    system "python2.7 setup.py build"
+    system "python3 setup.py build"
+  end
+
+  def self.install
+    system "python2.7 setup.py install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
+    system "python3 setup.py install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
```

Comment:
I'm glad you added this package.  There are several other packages that could be updated to use this as a dependency.  Also might be worth making a `setuptools.rb` package as well.  Something to think about.

Reply:
That's actually what I based this package off of originally. I will make another PR.